src/components/Sidebar/CampaignCard.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { CustomerConsumer } from "../CustomerContext";
+
+const CampaignCard = ({ campaign, code }) => {
+  const [expand, setExpand] = useState(false);
```

Comment:
`expanded` & `setExpanded` ?

`expand` sounds like some sort of action that we can trigger, but it's nothing more than a state

Reply:
It expands the Campaing card. On default you only see the campaing name. Then when you click on it, you can see more campaign data.